internal/manifest/os.go

```diff
@@ -176,14 +193,30 @@ func (p *OS) getPackageSetChain() []rpmmd.PackageSet {
 
 func (p *OS) getBuildPackages() []string {
 	packages := p.platform.GetBuildPackages()
+	if p.PartitionTable != nil {
+		packages = append(packages, p.PartitionTable.GetBuildPackages()...)
+	}
 	packages = append(packages, "rpm")
 	if p.OSTreeRef != "" {
 		packages = append(packages, "rpm-ostree")
 	}
 	if p.SElinux != "" {
-		packages = append(packages, "policycoreutils")
-		packages = append(packages, fmt.Sprintf("selinux-policy-%s", p.SElinux))
+		packages = append(packages, "policycoreutils", fmt.Sprintf("selinux-policy-%s", p.SElinux))
```

Comment:
Note that this is actually wrong (https://github.com/osbuild/osbuild-composer/issues/2954). We should take the policy specified in the image configuration into account. I think that those two values should be eventually merged and that's actually the plan (as to move away from osbuild-specific data structures). Just pointing out that this currently uses hardcoded value.

Reply:
ah good to know, thanks.  I was just merging the two lines here, but good that you noticed so we can fix it now if it's not too big a change.

Reply:
I think that it is fine to use this as is for the first rework, but we should make sure to not introduce any regressions in RHEL images before the development window for 9.2 and 8.8 closes...